Repository/CustomItemRepository.php

```diff
@@ -78,6 +78,21 @@ public function getTableAlias(): string
         return CustomItem::TABLE_ALIAS;
     }
 
+    /**
+     * @throws \Doctrine\ORM\NonUniqueResultException
+     * @throws \Doctrine\ORM\NoResultException
+     */
+    public function getItemCount($customObjectId): int
+    {
+        $queryBuilder = $this->getEntityManager()->createQueryBuilder();
+        $queryBuilder->select('count('.CustomItem::TABLE_ALIAS.'.id)');
+        $queryBuilder->from(CustomItem::class, CustomItem::TABLE_ALIAS);
+        $queryBuilder->where(CustomItem::TABLE_ALIAS.'.customObject = :customObjectId');
+        $queryBuilder->setParameter('customObjectId', $customObjectId);
+
```

Comment:
set limit 1 if you are using this function only for checking record exist or not.